aten/src/ATen/native/native_functions.yaml

```diff
@@ -6690,6 +6690,17 @@
   dispatch:
     CPU, CUDA: triangular_solve
 
+- func: linalg_solve_triangular.out(Tensor self, Tensor B, *, bool upper, bool left=True, bool unitriangular=False, Tensor(a!) out) -> Tensor(a!)
```

Comment:
In the future we may want to add additional kwargs for SciPy compat (like a unit_diagonal alias for unitriangular)